### DISCUSSION TITLE
Can you please add a method which can check if file have temporaryUrl or not?

### DISCUSSION THREAD

**Discussion by SushilMore1988** [livewire/livewire#5056] [3. Feature ideas]
Currently we are doing it by checking their extension type. also we have too much messy code in our livewire blade file. I tried to improve this but it is not working? if you can add method which can check if file has temporaryUrl or not then it will be just one if condition. Thanks anyway.

**Comment by joshhanley** [comment]
@SushilMore1988 if you have a cleaner way to handle that, PR's are welcome 🙂
